widgets/blueprints/src/ruleFilterFromString.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    CommonRuleOperator,
+    FilterRuleOperator,
+    FilterRuleType,
+    LabelsOnlyRuleOperator
+} from '../../common/src/filters/types';
+
+const OPERATOR_MAPPING: Record<string, any> = {
```

Comment:
I think we should have it named using lower camel case:
```suggestion
const operatorMapping: Record<string, any> = {
```

Ref.: https://github.com/airbnb/javascript#naming--uppercase